README.md

```diff
@@ -0,0 +1,151 @@
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+# 📖 covid19-worldometer <a name="about-project"></a>
+
+**covid19-worldometer** is a showcase for books.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Build With</summary>
+  <ul>
+    <li><a href="https://html.com/">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+    <li><a href="https://www.javascript.com/">JavaScript</a></li>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="add components"></a>
+
+- Showcase covid 19 statistics
+- Show details stats about covid
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo](https://covid19-worldometer.onrender.com) 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Prerequisites
+
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd math-magicians
+  git clone https://github.com/gsmalik030/covid19-worldometer
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd math-magicians
+  git clone https://github.com/gsmalik030/covid19-worldometer
+  npm install
+```
+
+### Usage
+
+Use this app by runing:
+
+```sh
+   cd math-magicians
+  git clone https://github.com/gsmalik030/covid19-worldometer
+  npm install
+  npm start
+```
+
+### Run tests
+
+Test this app by runing:
+
+```sh
+  cd math-magicians
+  git clone https://github.com/gsmalik030/covid19-worldometer
+  npm install
+  npm test
+```
+
+### Deployment
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Author <a name="authors"></a>
+
+👤 **Ghulam Subbhani(Malik)**
+
+- GitHub: [@githubhandle](https://github.com/gsmalik030)
+- Twitter: [@twitterhandle](https://twitter.com/gsmalik030)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/ghulam-subbhani-4b1281252/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Use react hooks**
+- [ ] **Implement full website**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/gsmalik030/covid19-worldometer/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, please, give a star.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank to all my Microverse partners.
```

Comment:
- [ ] Great job for adding some acknowledgments but I think it would be great if you give appropriate credit to the author. Therefore, you must do it in the README of your project so please add it there 🔥💪🏻🤩 as required 

###  You can add it like this 🔥💯
> The [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/), Original design idea by [Nelson Sakwa on Behance](https://www.behance.net/sakwadesignstudio).



![image](https://user-images.githubusercontent.com/58553711/225321993-2eedf67c-5ba8-41c6-bebc-57d25029f9ac.png)